request-manager/grapher/spec.go

```diff
@@ -33,6 +33,7 @@ type SequenceSpec struct {
 	Name  string               `yaml:"name"`  // name of the sequence
 	Args  SequenceArgs         `yaml:"args"`  // arguments to the sequence
 	Nodes map[string]*NodeSpec `yaml:"nodes"` // list of nodes that are a part of the sequence
+	Retry uint                 `yaml:"retry"` // the number of times to retry a "sequence" that fails
```

Comment:
Please excuse the minor non-code nit but why is "sequence" quoted? Is not a real sequence, or the same type of sequence previous fields refer to?

Reply:
It was copy/pasta from code comment above that says

```
// the number of times to retry a "job" that fails```

I'll remove the quotes